pkg/apis/config/feature_flags.go

```diff
@@ -118,7 +118,7 @@ func NewFeatureFlagsFromMap(cfgMap map[string]string) (*FeatureFlags, error) {
 }
 
 // setEnabledAPIFields sets the "enable-api-fields" flag based on the content of a given map.
-// If the feature gate is invalid or missing then an error is returned.
+// If the feature gate is invalid or missing then the flag is set to its default.
```

Comment:
NIT: this is not true, it does return error if the value is anything outside of `stable` and `alpha`, some git weirdness?

Reply:
Ah 😅 result of a bad merge. Good catch, thank you!